### PR TITLE
Dev/refactor project structure

### PR DESCRIPTION
**Pull request by ASVincent**
- Moves it from service based division to subnet/infrastructure grouping division.

### CODE REVIEW

File: modules/database_layer/variables.tf [hunk 1]
```diff
@@ -86,7 +86,41 @@ variable "db_port_num" {
...
+#--------------------------------------------------------------
+
+variable "ssm_prefix" {
+  description = "If set, saves db parameters to the parameter store with the given prefix"
+  default     = ""
+}
+
+variable "ssm_overwrite" {
+  description = "If true any existing parameters will be overwritten"
+  default     = "false"
```
Comment (tom-butler) [inline modules/database_layer/variables.tf]: Now that SSM has versioning, should this be true?

File: modules/ec2_instances/variables.tf [hunk 2]
```diff
@@ -1,91 +1,120 @@
...
+# Tags
+#--------------------------------------------------------------
+
 variable "owner" {}
+
+variable "cluster" {
+  description = "The name of the cluster"
+}
+
+variable "environment" {
```
Comment (tom-butler) [inline modules/ec2_instances/variables.tf]: should this be workspace to meet tf standards?

File: terraform.tfvars [hunk 3]
```diff
@@ -20,7 +20,9 @@ instance_type = "t2.small"
 
 ecs_aws_ami = "ami-c1a6bda2"
 
-owner = "UNKNOWN"
+owner = "YOUR EMAIL HERE"
 
 enable_jumpbox = false
 
+key_name = "YOUR SSH KEY HERE"
```
Comment (tom-butler) [inline terraform.tfvars]: Maybe specify this is the keyname, not the value of the key.

File: modules/ec2_instances/variables.tf (hunk 2 above)
Comment (ASVincent) [reply modules/ec2_instances/variables.tf]: Done :)

File: modules/database_layer/variables.tf (hunk 1 above)
Comment (ASVincent) [reply modules/database_layer/variables.tf]: I was hesitant because I haven't set up chamber yet, but will update it for the pattern :)

**Review by tom-butler — APPROVED** [review]

**Pull request merged**